package.json

```diff
@@ -21,5 +22,8 @@
     "eslint-plugin-node": "^5.2.1",
     "eslint-plugin-promise": "^3.6.0",
     "eslint-plugin-standard": "^3.0.1"
+  },
+  "bin": {
+    "apos-code-upgrader": "./bin/apos-code-upgrader"
```

Comment:
Keeping an apos- prefix on this for consistency with our new CLI's convenient name and to acknowledge we can't just own something as generic as `code-upgrader` at the shell prompt.